res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1334,32 +1336,17 @@ NumarkMixtrack3.BrowseButton = function(channel, control, value, status, group)
     var expand;
     var contract;
 
-    switch (skin){
-    case 1: // Deere or Shade
-    LibraryGroup = "[Master]";
-    LibraryCommand = "maximize_library";
-    expand = 1;
-    contract = 0;
-    break;
-
-    case 2: // LateNight
-    LibraryGroup = "[Master]";
-    LibraryCommand = "maximize_library";
-    expand = 1;
-    contract = 0;
-    break;
-
-    case 3: //Dark Metal
-    LibraryGroup = "[Hifi]";
-    LibraryCommand = "show";
-    expand = 0;
-    contract = 1;
-    break;  
-
-    default : 
-    LibraryGroup ="[Master]";
-    LibraryCommand = "maximize_library";
-    }
+    if (DarkMetalSkin){
```

Comment:
The alignment is off on GitHub's web rendering of this file. It seems that's because you've mixed tabs and spaces here. The rest of the file correctly uses 4 spaces to indent. Use that here too.

Also, IMO it's clearer to read to put a space between `)` and `{` for conditional statements. It looks like there are a few places that you didn't put a space between `if`/`for` and an opening `(` too. Some simple find and replace operations should take care of those.